tests/iio_info.c

```diff
@@ -248,7 +248,15 @@ int main(int argc, char **argv)
 				if (format->is_fully_defined)
 					sign += 'A' - 'a';
 
-				printf(", index: %lu, format: %ce:%c%u/%u>>%u)\n",
+				if (format->repeat)
+					printf(", index: %lu, format: %ce:%c%u/%uX%u>>%u)\n",
```

Comment:
I wonder if this should this be format->repeat > 1 so we get consistent output if there is no repeat.